### PR TITLE
LMAE TestFlight config

### PR DESCRIPTION
**Pull request by felix-u**
Add TestFlight config for LMAE.

### CODE REVIEW

File: GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg [hunk 1]
```diff
@@ -1,101 +1,116 @@
...
-//      FASA
-//      Squad
-//  ==================================================
-
-@PART[*]:HAS[#engineType[LMAE]]:FOR[RealismOverhaulEngines]
-{
-	%title = Lunar Module Ascent Engine (LMAE)
-	%manufacturer = Bell / Rocketdyne
-	%description = Pressure-fed engine used for the ascent module of the Apollo lunar lander.
-  @mass = 0.0816
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg:26]: If you hide whitespace changes, this becomes readable at least.

Why did you remove the patch to mass?

Comment (Capkirk123) [reply GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg:26]: The patch to mass does nothing, as MEC will overwrite any mass changes to its own settings.

Comment (DRVeyl) [reply GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg:26]: Alright, then.  @Capkirk123 , you good with the rest of the patch?

File: GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg [hunk 2]
```diff
@@ -1,101 +1,116 @@
...
+		@useEngineResponseSpeed = False
+		@engineAccelerationSpeed = 0
+		@engineDecelerationSpeed = 0
+		@useThrustCurve = False
+		%ullage = True
+		%pressureFed = True
+		%ignitions = 0
+
+		!IGNITOR_RESOURCE,*{}
+
```
[the remark this answers is not in the document]
Comment (felix-u) [reply GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg]: Removed it!

**Comment by DRVeyl** [comment]
Since merging the mass of engine config formatting, this conflicts.  We get a new chance to update the patch, hopefully without convincing GitHub that the entire file is different!  Can you re-apply it to the new version in the master branch, and keep with the formatting there?
